src/editorialreview/views.py

```diff
@@ -19,233 +19,230 @@
 
 @is_editor
 def add_editorial_review(request, submission_type, submission_id):
-	check = None
```

Comment:
Realised that I didn't actually make any changes to this file, just had to change indents when I temporarily put in a print statement, so nothing to check